scripts/driverkit/config/e5c53d648f3c4694385bbe488e7d47eaa36c229a/amazonlinux2_4.14.101-91.76.amzn2.x86_64_1.yaml

```diff
@@ -0,0 +1,6 @@
+kernelversion: 1
+kernelrelease: 4.14.101-91.76.amzn2.x86_64
+target: amazonlinux2
+output:
+  module: output/b7eb0dd65226a8dc254d228c8d950d07bf3521d2/scap_amazonlinux2_4.14.101-91.76.amzn2.x86_64_1.ko
```

Comment:
Does not appear to match the directory hash `e5c53d648f3c4694385bbe488e7d47eaa36c229a` 🤔

Reply:
oh god, i always forgot that...thanks!